docker-compose.yaml

```diff
@@ -18,6 +18,7 @@ services:
         condition: service_healthy
   web:
     image: mediacms/mediacms:latest
+    build: .
```

Comment:
We should not be changing this docker-compose, it's used for production. Create another which uses the image (with test artifacts) mentioned above